docs/reference/commandline/dockerd.md

```diff
@@ -64,6 +64,7 @@ Options:
       --raw-logs                              Full timestamps without ANSI coloring
       --registry-mirror value                 Preferred Docker registry mirror (default [])
       --selinux-enabled                       Enable selinux support
+      --shutdown-timeout=15                   Set the shutdown timeout value in seconds
```

Comment:
don't forget to update the man page as well https://github.com/docker/docker/blob/master/man/dockerd.8.md